src/mono/browser/runtime/pthreads/index.ts

```diff
@@ -9,8 +9,8 @@ export {
     mono_wasm_pthread_ptr, update_thread_info, isMonoThreadMessage, monoThreadInfo,
 } from "./shared";
 export {
-    mono_wasm_dump_threads, cancelThreads, is_thread_available,
```

Comment:
Was `is_thread_available` bad? Or is the reason for removal that it's not used? I am just curious why we don't keep `availableThreadCount` etc.

Reply:
it was not used anymore